src/Queue/TaskFinder.php

```diff
@@ -73,10 +70,8 @@ public function all(): array {
 
 	/**
 	 * @phpstan-return array<string, class-string<\Queue\Queue\Task>>
-	 *
```

Comment:
can we keep the noise commits minimal? I think maybe we could rather enforce the whitespace here instead of removing it, for readability.

Reply:
Well as explained above this was done automatically by the cakephp codesniffer since this is the CS ruleset we use.
If this is a rule this plugin enforces differently, it should be part of the phpcs.xml instead of having to manually take care of that.
I'll try to find that rule and add it to the phpcs.xml